MdeModulePkg/Test/Library/MockBaseResetSystemLibNull/MockBaseResetSystemLibNull.inf

```diff
@@ -0,0 +1,32 @@
+## @file
+#  Mocked library instance for ResetSystem library class
+#
+#  Copyright (c) 2006 - 2018, Intel Corporation. All rights reserved.<BR>
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = MockBaseResetSystemLibNull
+  FILE_GUID                      = FCDB0F20-B02A-4C5E-892A-7BF098B04CD9

```

Comment:
Let's change the GUIDs when moving them across repos, it should help avoid clashing with an older repo version.